src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -1821,6 +1821,14 @@ internal bool MightContainNoPiaLocalTypes()
         // compilation, while the other method needs a bindings object to determine what bound node
         // an expression syntax binds to.  Perhaps when we document these methods we should explain
         // where a user can find the other.
+        /// <summary>
+        /// Classifies a conversion from <paramref name="source"/> to <paramref name="destination"/> according
+        /// C# programming language.
```

Comment:
This wasn't quite what I meant. `Classifies a conversion from <paramref name="source"/> to <paramref name="destination"/>.`

Reply:
I understood. Late, but understood.